forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -62,12 +62,12 @@ class DFEdata:
     qubits: List[int]
     """qubits involved in the experiment"""
 
-    type: str
-    """Type of fidelity to be estimate (state or process)"""
+    kind: str
+    """Kind of fidelity to be estimate (exhaustive or monte carlo, state or process)"""
```

Comment:
fidelity to estimate, 
fidelity estimated

Reply:
Fixed